topics/first-tech-challenge/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: ftc, first-ftc, ftc-robotics, ftc-sdk
```

Comment:
https://github.com/topics/ftc-robotics doesn't list any repositories right now. Since the topic isn't in use on any public repos, can you remove it from this line? I do see https://github.com/topics/ftc-robot-controller and https://github.com/topics/first-robotics-competition have repositories, though, if those would make sense to you as aliases.

Reply:
`first-robotics-competition` is a different topic. FIRST runs two competitions - FIRST Tech Challenge (FTC) which is for students in grades 7-12, where the robots are smaller and it costs less to run a team. FIRST Robotics Competition (FRC) is a competition for 9-12th graders, and the robots are much larger.

`ftc-robot-controller` makes sense, and I'll add it, as well as take out `ftc-robotics`. Thanks for the suggestion!

Reply:
@cheshire137 I added `ftc-robot-controller` and removed `ftc-robotics`.

`first-robotics-competition` is an entirely different competition. FIRST runs two competitions - FTC, which is for students in grades 7-12, and FIRST Robotics Competition (FRC), which is for 9-12th graders and has much larger robots.